File: LeetCode/Easy/0412-fizz-buzz/0412-fizz-buzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        ArrayList<String> ret = new ArrayList<>();
        for (int i = 1; i < n + 1; ++i){
            if (i % 5 == 0 && i % 3 == 0) ret.add("FizzBuzz");
            else if (i % 3 == 0) ret.add("Fizz");
            else if (i % 5 == 0) ret.add("Buzz");
            else ret.add(String.valueOf(i));
        }
        return ret;
    }
}

File: LeetCode/Easy/0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ret = []
        for i in range(1, n+1):
            if i % 3 == 0 and i % 5 == 0:
                ret.append("FizzBuzz")
            elif i % 3 == 0:
                ret.append("Fizz")
            elif i % 5 == 0:
                ret.append("Buzz")
            else:
                ret.append(str(i))
        return ret